packages/devtools/pubspec.yaml

```diff
@@ -9,7 +9,7 @@ description: A suite of web-based performance tooling for Dart and Flutter.
 # package remaining purely as a container for the compiled copy of the devtools
 # app. That ensures that version constraints for the devtools_app do not impact
 # this package.
-version: 0.1.8
+version: 0.1.9
```

Comment:
if this change does need to be published right away, please update the version numbers in the files specified by the above code comments.